leaflet-iiif.js

```diff
@@ -165,7 +163,66 @@ L.TileLayer.Iiif = L.TileLayer.extend({
 
         // Resolved Deferred to initiate tilelayer load
         _this._infoDeferred.resolve();
-      });
+      };
+
+      var onload = function(rsp){
+
+                var target = rsp.target;
+
+                if (target.readyState != 4){
+                    return;
+                }
+
+                var status_code = target['status'];
+                var status_text = target['statusText'];
+
+                var raw = target['responseText'];
+                var data = undefined;
+
+                try {
+                    data = JSON.parse(raw);
+                }
+
+                catch (e){
+
+                    console.log("Failed to parse JSON, because " + e);
+		    console.log(raw);
```

Comment:
Looks like there is a mix of tabs and four spaces. Mind converting over to two spaces?